Sources/AWSLambdaRuntime/LambdaRuntime.swift

```diff
@@ -39,7 +40,7 @@ public final class LambdaRuntime<Handler>: @unchecked Sendable where Handler: St
         eventLoop: EventLoop = Lambda.defaultEventLoop,
         logger: Logger = Logger(label: "LambdaRuntime")
     ) {
-        self.handlerMutex = NIOLockedValueBox(handler)
+        self.handlerMutex = SendingStorage(handler)
```

Comment:
Its not really a Mutex now
```suggestion
        self.handlerStorage = SendingStorage(handler)
```